types/call-bind/test/callBind.test.ts

```diff
@@ -0,0 +1,19 @@
+import callBind = require('call-bind');
+import { apply as applyBind } from 'call-bind';
+
+declare const any: unknown;
```

Comment:
why this instead of just using `unknown` directly?

Reply:
Because `unknown` only refers to a type.

Reply:
gotcha, so it could be named anything then. i was just confused by the name `any`

Reply:
Agree, `any` is a super confusing name for a value of type `unknown`. Maybe try `unknown` ?